Sources/Runner/main.swift

```diff
@@ -1,9 +1,12 @@
+import Backtrace
 import Either
 import Foundation
 import PointFree
 
 // Bootstrap
 
+Backtrace.install()
```

Comment:
I guess this could also be a step in the `bootstrap` process? think that's worth it?

Reply:
I thought about that but liked the explicitness of it being called in `main.swift` as a dependency. Don't feel super strongly about it, though. Want me to move?